javaparser-symbol-solver-core/src/main/java/com/github/javaparser/symbolsolver/utils/SymbolSolverCollectionStrategy.java

```diff
@@ -6,22 +6,16 @@
 import com.github.javaparser.symbolsolver.resolution.typesolvers.JarTypeSolver;
 import com.github.javaparser.symbolsolver.resolution.typesolvers.JavaParserTypeSolver;
 import com.github.javaparser.symbolsolver.resolution.typesolvers.ReflectionTypeSolver;
-import com.github.javaparser.utils.CollectionStrategy;
-import com.github.javaparser.utils.Log;
+import com.github.javaparser.utils.ParserCollectionStrategy;
 import com.github.javaparser.utils.ProjectRoot;
 
 import java.io.IOException;
-import java.nio.file.*;
-import java.nio.file.attribute.BasicFileAttributes;
-
-import static java.nio.file.FileVisitResult.CONTINUE;
-import static java.nio.file.FileVisitResult.SKIP_SUBTREE;
+import java.nio.file.Path;
 
 /**
- * Strategy which collects all SourceRoots and initialises the TypeSolver and returns the SourceRoots configured
- * with the TypeSolver in a ProjectRoot object.
+ * Attempts to configure the symbol solver to look at the source code and jar files found in a project directory.
  */
-public class SymbolSolverCollectionStrategy implements CollectionStrategy {
+public class SymbolSolverCollectionStrategy {
```

Comment:
This thing has become really badly named now.

Reply:
Well, then rename it, removing the interface is already a backwards-incompatible change. :-)